docs/toc.md

```diff
@@ -1,164 +1,93 @@
-#### [dotnet-publish](core-concepts/core-sdk/cli/dotnet-publish.md)
-#### [dotnet-install-script](core-concepts/core-sdk/cli/dotnet-install-script.md)
-## [🔧 .NET Core Servicing](core-concepts/servicing.md)
-## [Developing with .NET Core](core-concepts/development.md)
-### [Web applications](core-concepts/web-apps/index.md)
-### [Console applications](core-concepts/console/index.md)
-#### [🔧 Overview](core-concepts/console/overview.md)
-#### [🔧 Writing Console Apps: An advanced step-by-step guide](core-concepts/console/cli-console-app-tutorial-advanced.md)
-### [Libraries](core-concepts/libraries/index.md)
-#### [Writing Libraries with Cross Platform Tools](core-concepts/libraries/libraries-with-cli.md)
-#### [🔧 Creating Libraries in Visual Studio](core-concepts/libraries/libraries-with-vs.md)
-### [Testing](core-concepts/testing/unit-testing.md)
-#### [Unit Testing with dotnet test](core-concepts/testing/unit-testing-with-dotnet-test.md)
-## [🔧 .NET Core Application Deployment](core-concepts/app-deployment.md)
-## [Migrating from DNX](core-concepts/dnx-migration.md)
-## [Porting applications between editions of .NET Platform](porting/index.md)
-### [🔧 Overview](porting/overview.md)
-### [Tooling for helping the porting](porting/tooling.md)
-### [🔧 Porting your applications](porting/applications.md)
-### [🔧 Porting your libraries](porting/libraries.md)
-### [🔧 Porting NuGet packages](porting/nuget-packages.md)
+## [Deployment](scenarios/deployment.md)
+### [Using .NET Core in Continuous Integration](scenarios/using-core-with-ci.md)
+### [Using .NET Core from a USB stick](scenarios/using-core-usb.md)
+### [Using .NET Core with no Internet](scenarios/using-core-offline.md)
+### [Deploying a .NET Core app with Docker](scenarios/deploying-core-with-docker.md)
+## [Azure](scenarios/azure.md)
+### [Using Blob Storage for dynamic data](scenarios/blob-storage.md)
+### [Using LINQ with Azure Table](scenarios/linq-azure-table.md)
+### [Using a Console app as a WebJob](scenarios/console-web-worker.md)
+### [Using Azure as a backend for a Xamarin app](scenarios/xamarin-azure-backend.md)
+### [Using SQL Azure with .NET Core](scenarios/sql-azure-core.md)
+## [Games](scenarios/games.md)
+### [Writing a VR app with Unity](scenarios/vr-unity.md)
+## [Porting code between .NET Platforms](scenarios/porting.md)
+# [C# Guide](csharp/index.md)
+## [About C#](csharp/about/index.md)
+### [Community](csharp/about/community.md)
+### [Open Source](csharp/about/opensource.md)
+### [Learning Resources](csharp/about/learn.md)
+## [Releases](csharp/releases.md)
+### [C# 7](csharp/7.md)
+### [C# 6](csharp/6.md)
+## [Handbook](csharp/handbook.md)
+## [C# Editors](csharp/editors.md)
+## [C# Console Application Tutorial](csharp/console-teleprompter.md)
+# [F# Guide](fsharp/index.html)
+## [Community](fsharp/community.md)
+## [Open Source](fsharp/open-source.md)
+## [Learning Resources](fsharp/learn.md)
+## [Talks](fsharp/talks.md)
+# [.NET Core Guide](core/index.md)
```

Comment:
Right now, this section seems a bit of a jumble. I think 3 levels would help. Suggested 2nd level would be:

## Supported Operating Systems

## Programming Languages

## Application Types (I want a better phrase here. I'm looking for a heading for "applications", "libraries" "using multiple projects", "asp.net applications", and "unit testing".

## Deploying and Versioning